components/exchange/Translation.js

```diff
@@ -28,6 +28,11 @@ const WelcomeText = styled.p`
   light {
     font-weight: 600;
   }
+  desktop {
+    @media screen and (max-width: ${screenSizes.smallPhone.max}) {
+      display: none;
+    }
+  }
```

Comment:
This is a great solution, and very clean! I do think it is better to use real html-tags cause we might get semantic problems otherwise. You could do something like this:
```suggestion
  .desktop {
    @media screen and (max-width: ${screenSizes.smallPhone.max}) {
      display: none;
    }
  }
```

and then use it like 
```
<div className="desktop">
...
</div>
```

Reply:
I guess this goes for `light` as well

Reply:
Done (Y)